### ISSUE TITLE
2.5.5 Understanding: Less examples in WCAG 2.2 than 2.1

### ISSUE THREAD

**Issue by johannesFischer84** [w3c/wcag#3617]
In the [Understanding document of SC 2.5.5 (Target Size), WCAG 2.1](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html) there are seven examples, e. g. "Example 7: Help icon". In the [same SC of WCAG 2.2 (Target Size Enhanced)](https://www.w3.org/WAI/WCAG22/Understanding/target-size-enhanced.html) there is only 1 example left: "Example 1: Buttons".

Does this mean the other examples are not valid anymore? Or is the loss of examples a bug in WCAG 2.2?

**Comment by patrickhlauke** [comment]
Looks like whenever the 2.2 version was set up, some of the content from the 2.1 version was dropped (seemingly accidentally)

**Comment by giacomo-petri** [comment]
During the review process, along with the examples, also the purpose of the exceptions has been overlooked. For instance:
> Equivalent targets: If there is more than one target on a screen that performs the same action, only one of the targets need to meet the target size of 44 by 44 CSS pixels.
> 
> Inline: Content displayed can often be reflowed based on the screen width available. This is known as responsive design and makes it easier to read since you do not need to scroll both horizontally and vertically...
is no longer available in 2.2 understanding doc.

I will align version 2.2 with version 2.1, and subsequently, we can deliberate on whether there are any elements that should be omitted.
